
From: Matthew Dobson <colpatch@us.ibm.com>

Here's a small comment fix for include/linux/nodemask.h.  It's a comment
that is only relevant to cpumasks and accidentally got copied over during
my (failed) attempt to correctly post the nodemask code.  The patch just
updates the comment to make sense for nodemasks.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/nodemask.h |    9 +++------
 1 files changed, 3 insertions(+), 6 deletions(-)

diff -puN include/linux/nodemask.h~fix-comment-in-include-linux-nodemaskh include/linux/nodemask.h
--- 25/include/linux/nodemask.h~fix-comment-in-include-linux-nodemaskh	Thu Aug 26 16:38:51 2004
+++ 25-akpm/include/linux/nodemask.h	Thu Aug 26 16:38:51 2004
@@ -69,12 +69,9 @@
  *
  * Subtlety:
  * 1) The 'type-checked' form of node_isset() causes gcc (3.3.2, anyway)
- *    to generate slightly worse code.  Note for example the additional
- *    40 lines of assembly code compiling the "for each possible node"
- *    loops buried in the disk_stat_read() macros calls when compiling
- *    drivers/block/genhd.c (arch i386, CONFIG_SMP=y).  So use a simple
- *    one-line #define for node_isset(), instead of wrapping an inline
- *    inside a macro, the way we do the other calls.
+ *    to generate slightly worse code.  So use a simple one-line #define
+ *    for node_isset(), instead of wrapping an inline inside a macro, the
+ *    way we do the other calls.
  */
 
 #include <linux/kernel.h>
_
